k8s/data/broker.go

```diff
@@ -0,0 +1,32 @@
+package data
+
+import (
+	"k8s.io/client-go/1.4/pkg/api/unversioned"
+	"k8s.io/client-go/1.4/pkg/api/v1"
+)
+
+type BrokerState string
+
+const (
+	BrokerStatePending   BrokerState = "Pending"
+	BrokerStateAvailable BrokerState = "Available"
+	BrokerStateFailed    BrokerState = "Failed"
+)
+
```

Comment:
For my own edification, what are the conditions that lead a broker to be in one of these three states? If this resource represents / references another piece of software (a "backing broker") then aren't there really only two states a backing broker can be in (in relation to Steward at least)? Available / operational or not?